<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTime</class>
 <widget class="QWidget" name="DateTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="CustomTimeCtrl.qrc">
    <normaloff>:/agenda.png</normaloff>:/agenda.png</iconset>
  </property>
  <property name="statusTip">
   <string>Date Time Input</string>
  </property>
  <property name="whatsThis">
   <string>Widget to Input DateTime</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupDate">
            <property name="toolTip">
             <string>Input the date here</string>
            </property>
            <property name="statusTip">
             <string>Input date</string>
            </property>
            <property name="whatsThis">
             <string>Widget to input date</string>
            </property>
            <property name="title">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="lbDate">
               <property name="text">
                <string>Date</string>
               </property>
               <property name="buddy">
                <cstring>dateEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDateEdit" name="dateEdit">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="lbDateFormat">
               <property name="text">
                <string>-&gt; DATE FORMAT</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupTime">
            <property name="toolTip">
             <string>Input the time here</string>
            </property>
            <property name="statusTip">
             <string>Input time</string>
            </property>
            <property name="whatsThis">
             <string>Widget to input time</string>
            </property>
            <property name="title">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="lbTime">
               <property name="text">
                <string>Time</string>
               </property>
               <property name="buddy">
                <cstring>timeEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QTimeEdit" name="timeEdit">
               <property name="calendarPopup">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="lbTimeFormat">
               <property name="text">
                <string>-&gt; TIME FORMAT</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="checkTime">
          <property name="text">
           <string>Has time</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="CustomTimeCtrl.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>DateTime</receiver>
   <slot>setHasTime(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setHasTime(bool)</slot>
 </slots>
</ui>
